qgreenland/tasks/common/misc.py

```diff
@@ -13,6 +13,13 @@
 
 class Decompress(LayerTask):
     task_type = TaskType.WIP
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if 'decompress_kwargs' in self.layer_cfg:
+            self.decompress_kwargs = self.layer_cfg['decompress_kwargs']
+        else:
+            self.decompress_kwargs = {}
```

Comment:
We've done this elsewhere:

```
kwargs.update(cfg.get('ingest_task_kwargs', {}))
```

We could do:

```
self.decompress_kwargs = self.layer_cfg.get('decompress_kwargs', {})
```

and eliminate the conditional.

Reply:
Good call :+1: